src/neo/SmartContract/Manifest/ContractManifest.cs

```diff
@@ -171,9 +180,9 @@ private void DeserializeFromJson(JObject json)
             Groups = ((JArray)json["groups"]).Select(u => ContractGroup.FromJson(u)).ToArray();
             Features = ContractFeatures.NoProperty;
             Permissions = ((JArray)json["permissions"]).Select(u => ContractPermission.FromJson(u)).ToArray();
-            Trusts = WildCardContainer<UInt160>.FromJson(json["trusts"], u => UInt160.Parse(u.AsString()));
-            SafeMethods = WildCardContainer<string>.FromJson(json["safeMethods"], u => u.AsString());
-
+            Trusts = WildcardContainer<UInt160>.FromJson(json["trusts"], u => UInt160.Parse(u.AsString()));
+            SafeMethods = WildcardContainer<string>.FromJson(json["safeMethods"], u => u.AsString());
+            Extra = JObject.Parse(json["extra"].AsString());
```

Comment:
Max `extra` size?

Reply:
@shargon what limit do you think is reasonable? 2048?

Reply:
Also, do you think it makes sense to limit only a property and not the whole manifest? Maybe if we are adding a maximum size, it should be on the manifest (if not added already)

Reply:
Maybe we should limit whole the manifest in self?

Reply:
@shargon  in the manifest or in JsonSerialize, similar to what Erik previous PRs.  I think it makes more sense to verify the whole file or transaction instead of verifying each field individually.